PYME/Acquire/Hardware/pco/pco_cam.py

```diff
@@ -242,12 +237,13 @@ def SetROI(self, x0, y0, x1, y1):
 
         # Recording state is reset, so set to 0
         self.n_read = 0
-
-        self.SetDescription() # Update the description, for safety. TODO: Is this necessary??
+        self.GetROI()
 
     def GetROI(self):
-        roi = self.cam.sdk.get_roi()
-        return roi['x0'], roi['y0'], roi['x1'], roi['y1']
+        if self.n_read == 0:
```

Comment:
I worry a little about the `n_read` stuff - will merge for now as camera driver still very 'alpha', but feels like it might be fragile. On other cameras (uc480 in particular) we have just cached the ROI values that we set in `SetROI()`.

Reply:
Fair enough. I don't have a particularly good reason for the `self.n_read==0` call. My original goal was to just confirm that the ROI we set is exactly the ROI we get back. I think I've confirmed this and I can probably just store the set ROI in `__roi`.